src/components/header/Header.jsx

```diff
@@ -0,0 +1,55 @@
+import { Logo } from "../logo/Logo";
+
+export const Header = () => {
+  return (
+    <>
+      <header className="header">
+        <nav className="navbar">
+          <div className="nav-brand">
+            <button className="btn hamburger-menu">
+              <i className="fas fa-bars icon" />
+            </button>
+            <Logo variant={"header"} />
+          </div>
+          <div className="search">
+            <i className="fa-solid fa-magnifying-glass icon" />
+            <input
+              className="search-field"
+              type="search"
+              placeholder="Search..."
+              aria-label="Search Products"
+            />
+          </div>
+          <div className="nav mobile-nav">
+            <button className="btn nav-close-btn">
+              <i className="fas fa-arrow-left" />
+            </button>
+            <ul className="menu">
+              <li>
+                <a href="./pages/login.html" className="btn btn-primary">
```

Comment:
`Link` can be used instead of `a` to avoid page refresh

Reply:
Thanks for noticing, will fix it.